src/screens/General/About.js

```diff
@@ -1,76 +1,169 @@
+					<Text style = { [textColor, titleStyle, { fontSize: 20 }] }>Developed by:</Text>
+					<View>
+						<FlatList
+							numColumns = { 3 }
+							columnWrapperStyle = {{ justifyContent: "space-around" }}
+							data = { list.developers }
+							renderItem = { this.renderDev }
+							keyExtractor = { this.keyExtractor }
+						/>
+					</View>
+					<TouchableOpacity onPress = { () =>
+						Linking.openURL("https://github.com/SHPEUCF/shpeucfapp/graphs/contributors") }
+					>
+						<Text style = { [textColor, contributorStyle] }>
+							...and our amazing contributors.
+						</Text>
+					</TouchableOpacity>
+					<View style = { [center, footer] }>
+						<Text style = { textColor }>Copyright © 2018 SHPE UCF</Text>
+					</View>
+				</View>
 			</SafeAreaView>
 		);
 	}
 
 	keyExtractor = (item, index) => index
 
+	version = () => {
+		const {
+			textColor,
+			size
+		} = styles;
+
+		return (
+			<View style = {{ alignItems: "flex-end" }}>
+				<Text style = { [textColor, size] }>
+					v1.1
+				</Text>
+			</View>
+		);
+	}
+
 	renderItem = ({	item }) => {
+		const {
+			titleWeight,
+			bgColor,
+			textColor
+		} = styles;
+
 		return (
-			<View style = {{ backgroundColor: "black" }}>
+			<View>
 				<ListItem
-					containerStyle = {{ backgroundColor: "black", borderBottomColor: "black" }}
-					removeClippedSubviews = { false }
+					containerStyle = { bgColor }
+					onPress = { () => Linking.openURL(item.url) }
 					title = { item.title }
-					titleStyle = {{ color: "white" }}
-					leftIcon = {{ name: item.icon, color: "white" }}
+					titleStyle = { [titleWeight, textColor] }
+					subtitle = { item.content }
+					subtitleStyle = { textColor }
+					leftIcon = { <Ionicons
+						name = { item.icon }
+						size = { 26 }
+						color = { "#FFF" }
+					/> }
 					rightIcon = { <Ionicons
 						name = "ios-arrow-dropright"
 						size = { dimension.height * 0.025 }
-						style = {{ color: "#FECB00" }}
+						color = { "#FECB00" }
 					/> }
-					onPress = { () => Actions[item.screen]({
-						title: item.title,
-						uri: item.uri,
-						content: item.content,
-						footer: item.footer
-					}) }
 				/>
 			</View>
 		);
 	}
+
+	renderDev = ({ item }) => {
+		const {
+			center,
+			textColor,
+			textSize
+		} = styles;
+
+		let idx = item.name.indexOf(" ");
+		let firstName = item.name.substring(0, idx);
+		let lastName = item.name.substring(idx + 1);
+
+		return (
+			<TouchableOpacity onPress = { () => Linking.openURL(item.github) }>
+				<View style = { center }>
+					<Avatar
+						size = { dimension.height * 0.09 }
+						rounded
+						source = {{ uri: item.pic }}
+					/>
+				</View>
+				<View style = { center }>
+					<Text style = { [textColor, textSize] }>{ firstName }</Text>
+					<Text style = { [textColor, textSize] }>{ lastName }</Text>
+				</View>
+			</TouchableOpacity>
+		);
+	}
 }
 
+const styles = {
+	bgColor: {
+		backgroundColor: "#000"
+	},
+	textColor: {
```

Comment:
textColor and Size can probably be 1 style since we will want all text to be the same most of the time. And then you just override the size in places where you want it to be different.
```suggestion
	textStyling {
	   color:"#FFF"
	   fontSize: 16
	}
	
	style = {textStyling}
	style = {[textStyling, {fontSize: overridehere}]}
```